packages/react-composites/tests/browser/call/app/mocks/MockCallAdapter.ts

```diff
@@ -0,0 +1,237 @@
+  async leaveCall(forEveryone?: boolean): Promise<void> {
+    return;
+  }
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  async startCamera(options?: VideoStreamOptions): Promise<void> {
+    return;
+  }
+  async stopCamera(): Promise<void> {
+    return;
+  }
+  async mute(): Promise<void> {
+    return;
+  }
+  async unmute(): Promise<void> {
+    return;
+  }
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  startCall(participants: string[]): Call | undefined {
+    return;
+  }
+  async startScreenShare(): Promise<void> {
+    return;
+  }
+  async stopScreenShare(): Promise<void> {
+    return;
+  }
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  async removeParticipant(userId: string): Promise<void> {
+    return;
+  }
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  async createStreamView(remoteUserId?: string, options?: VideoStreamOptions): Promise<void> {
+    return;
+  }
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  async disposeStreamView(remoteUserId?: string, options?: VideoStreamOptions): Promise<void> {
+    return;
+  }
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  async askDevicePermission(constrain: PermissionConstraints): Promise<void> {
+    return;
+  }
+
+  public async queryCameras(): Promise<VideoDeviceInfo[]> {
+    return [];
+  }
+
+  public async queryMicrophones(): Promise<AudioDeviceInfo[]> {
+    return [];
+  }
+
+  public async querySpeakers(): Promise<AudioDeviceInfo[]> {
+    return [];
+  }
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  async setCamera(sourceInfo: VideoDeviceInfo, options?: VideoStreamOptions): Promise<void> {
+    return;
+  }
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  async setMicrophone(sourceInfo: AudioDeviceInfo): Promise<void> {
+    return;
+  }
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  async setSpeaker(sourceInfo: AudioDeviceInfo): Promise<void> {
+    return;
+  }
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars, @typescript-eslint/no-explicit-any
+  on(event: string): void {
+    return;
+  }
+
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars, @typescript-eslint/no-explicit-any
+  off(event: string): void {
+    return;
+  }
+}
+
+/**
+ * Helper function to create remoteParticipants property of CallAdapterState from an array of TestRemoteParticipant
+ * @param mockRemoteParticipants - array of TestRemoteParticipant
+ * @returns Record of RemoteParticipantState
+ */
+
+const createMockRemoteParticipants = (
+  mockRemoteParticipants: TestRemoteParticipant[]
+): Record<string, RemoteParticipantState> => {
+  const remoteParticipants: Record<string, RemoteParticipantState> = {};
+
+  // Incrementing participant key starting at 1
+  let participantKey = 1;
+
+  for (const remoteParticipant of mockRemoteParticipants) {
```

Comment:
nit: instead of using a key that incremenets on each loop, you can instead do:
```
  for (const [remoteParticipant, i] of mockRemoteParticipants) {
```